### PR TITLE
WIP Different CSS, and more

### PR DESCRIPTION
**Pull request by whippingdot**
This PR started as I wanted to change the CSS without deleting everything everyone made. But now we have done that and more so that is why the PR name is different. We just need to add a few more profiles and make a way for us to send the join requests to a DB/email, and we are done. Please people who haven't created a profile make one.

### CODE REVIEW

**Comment by whippingdot** [comment]
@programmeruser2 @Siddhant-K-code @SixBeeps @Explosion-Scratch @elipie @RecursiveDescent @firefish111 @forerosantiago @zplusfour @CosmoCat07 @AlphaMirage @AmazingMech2418 @CyanCoding @cursorweb @tickedoffsquid @rohilpatel1 @Isaiah08-D You guys still have to create ur profiles. A few more people I missed might be there so you also create it.

**Comment by whippingdot** [comment]
@BD103 we made ur page open in a new tab because it takes us to a different page + you haven't linked the original page from there

**Comment by zghwi** [comment]
I'll create it after school
But I'll fork it and then I'll create a pr, I dont wanna code on your repl.

**Comment by whippingdot** [comment]
lul fine. I thought you could finish your profile quick and then leave but...I also wanted you to do this as it would mean if there are any changes made on the main one we all are working on you won't have to pull when we commit or we won't have to pull when you commit. People also stopped annotating so...Just saying.

**Comment by zghwi** [comment]
No that's not the reason, I can't commit and push from repl, that's why I have a problem.
In my opinion vscode is better.

**Comment by forerosantiago** [comment]
@whippingdot I'm sorry... but where are the other websites?

**Comment by whippingdot** [comment]
> @whippingdot I'm sorry... but where are the other websites?

What do you mean?

**Comment by rohilpatel1** [comment]
> @programmeruser2 @Siddhant-K-code @SixBeeps @Explosion-Scratch @elipie @RecursiveDescent @firefish111 @forerosantiago @zplusfour @CosmoCat07 @AlphaMirage @AmazingMech2418 @CyanCoding @cursorweb @tickedoffsquid @rohilpatel1 @Isaiah08-D You guys still have to create ur profiles. A few more people I missed might be there so you also create it.

Wut?

**Comment by whippingdot** [comment]
Go to the repl i invited you to and finish making your part of the portfolio. You have to make ur own page.
https://repl.it/@Whippingdot/About-Page

**Comment by rohilpatel1** [comment]
> Go to the repl i invited you to and finish making your part of the portfolio. You have to make ur own page.
> https://repl.it/@Whippingdot/About-Page

404

**Comment by whippingdot** [comment]
Bruh in the repl you will see a file for everyone in our org:
image

You just have to add your own file and link it in the navbar of the signup and index pages.

P.S. It seems u weren't invited so I invited you again. Join the repl:
https://repl.it/@Whippingdot/About-Page

**Comment by firefish111** [comment]
bah

**Comment by whippingdot** [comment]
O ya firefish please make ur page already too

**Comment by firefish111** [comment]
bah

**Comment by whippingdot** [comment]
bRuH wOrK oN iT pUhLeAsE wIlL yOu i WaNt tO fInIsH iT. PuHlEaSe

File: views/whippingdot.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		<h2 class="center">Moi Friends</h2>
+		<p></p>
+
+		<h2 class="center">Basic Facts</h2>
+		<div class="center">
+			<ul>
+				<li>
+					Everyone who knows me closely hates me or thinks I am irritating
+				</li>
+				<li>I have bad convincing skills</li>
```
Comment (whippingdot) [inline views/whippingdot.html]: Welp, I don't have discord...yet

**Comment by whippingdot** [comment]
Please can I have at least one reviewer review these changes so I can merge. I can merge rn but I want to get the opinion of someone here. @BD103 @darkdarcool @JBYT27 @ch1ck3n-byte @firefish111 @AmazingMech2418 anyone please?????

**Comment by kokonut27** [comment]
> Please can I have at least one reviewer review these changes so I can merge. I can merge rn but I want to get the opinion of someone here. @BD103 @darkdarcool @JBYT27 @ch1ck3n-byte @firefish111 @AmazingMech2418 anyone please?????

Sure, just busy. Ill leave a comment in the repl

**Comment by whippingdot** [comment]
?? Foine (but I haven't gotten it yet)

**Comment by kokonut27** [comment]
Ik, ill do it later (busy)

**Comment by whippingdot** [comment]
k I won't disturb ya. I will try to improve my portfolio on VS Code until then...

**Comment by whippingdot** [comment]
Hey slipped my mind can anyone fix the 'ad' not working on my page

**Comment by Siddhant-K-code** [comment]
Why didn't we created a general template for the coder's portfolio.  It will look good.

**Comment by whippingdot** [comment]
I wanted everyone to have their own wacky porfolio (read the readme)

**Comment by kokonut27** [comment]
Hey @whippingdot, I added a suggestions.md. Not much, but I added my suggestion there

**Comment by whippingdot** [comment]
will add it bruh

**Comment by firefish111** [comment]
@whippingdot é

**Comment by firefish111** [comment]
you've been é-ed

**Comment by aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa4** [comment]
eed

**Comment by whippingdot** [comment]
Firefish is rogue. Also @firefish111 I let everyone have their own CSS on purpose. Ploise just join us and create ur page. PLOISE!

**Comment by whippingdot** [comment]
Oh and @SixBeeps u seem to be good in fixing the linking to stuff not working I think @ch1ck3n-byte is good too. If either of u can please fix the 'ad' not working on my page. I really need it lul

**Comment by firefish111** [comment]
> Firefish is rogue. Also @firefish111 I let everyone have their own CSS on purpose. Ploise just join us and create ur page. PLOISE!

well then i think already have created mine @whippingdot

**Comment by whippingdot** [comment]
huh what do you mean?

**Review by darkdarcool — COMMENTED** [review]
Looks good.

**Comment by whippingdot** [comment]
bruh i already merged lul

**Comment by darkdarcool** [comment]
ik

**Comment by darkdarcool** [comment]
just leaving the review it told me to put.

**Pull request merged**